prometheus-authguard.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Prometheus Authguard Server Instance
```

Comment:
mmh, yes. What Description do you prefer? `Authguard Server Instance`? I would prefer `Authguard Instance` or alternatively `Authguard Relay Instance`, because imho authguard is no server. It's a relay or something you put between your Webservice and the wideness of the internet.